docs/pages/connect-your-client/vnet.mdx

```diff
@@ -206,21 +223,47 @@ $ grep tsh /var/log/com.apple.xpc.launchd/launchd.log
 ```
 </TabItem>
 <TabItem label="Windows">
-Logs are saved to a custom log in Event Log called Teleport. To browse them, open [Event
-Viewer](https://learn.microsoft.com/en-us/shows/inside/event-viewer), select "Applications and
-Services Logs" in the sidebar on the left and choose "Teleport".
+To enable debug logs in VNet, first stop Teleport Connect. Then in the Start menu look for Command
+Prompt and from the right click menu select Run as administrator. The following command enables
+debug logs in VNet.
 
-To save them to a file, select "Save All Events As…" from the sidebar on the right.
+{/* `&& exit` at the end of the command is there on purpose so that the user doesn't start Connect
+from the admin command prompt. */}
```

Comment:
I would work the comment into the preceding paragraph so the user doesn't remove the `&& exit`, e.g.:

```
debug logs in VNet. The `&& exit` at the end of the command prevents you from starting Connect from the admin command prompt.
```